model/test/user.go

```diff
@@ -1,5 +1,7 @@
 package user
 
+//go:generate scaffold model
```

Comment:
I don't know what the pragmatic way of doing this is. I tried using it in my other project. I liked it this way because my other project doesn't even need scaffold as a dependency. As long as scaffold is on the path it runs. 

With that said anyone can write their own go:generate command however they want. 

I'd love to add a go:generate command when generating the code, but right now the models are generated into a single file and it's up to you to split them up. Gqlgen has a follow-schema option in main. Could work if we can spec a template as well.